archive_pipeline/README.md

```diff
@@ -0,0 +1,37 @@
+# Archive Pipeline
+
+This directory contains the code and supporting files for running the pipeline for removing old files from the database and archiving data in a AWS S3 Data Warehouse via .csv files.
```

Comment:
I think here maybe just clarify that we are saying old files are over 24 hours old